migration/types.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2023 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package migration
```

Comment:
Other packages call this file `interface.go`
Maybe consistency is best?

Reply:
Sure.